recipes/bcftools/1.10/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -ex
+
+n="$CPU_COUNT"
+
+CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" \
+        ./configure --prefix="$PREFIX" --with-htslib="$PREFIX" 
+
+make -j $n CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib"
```

Comment:
If `configure` is well behaved, the `CPPFLAGS` and `LDFLAGS` here are not needed.